spanner/transaction.go

```diff
@@ -373,7 +377,7 @@ func (t *txReadOnly) ReadRowWithOptions(ctx context.Context, table string, key K
 // ReadRowUsingIndex reads a single row from the database using an index.
 //
 // If no row is present with the given index, then ReadRowUsingIndex returns an
-// error where spanner.ErrCode(err) is codes.NotFound.
+// error(spanner.ErrRecordNotFound) where spanner.ErrCode(err) is codes.NotFound.
```

Comment:
Also here: Maybe add an example in the comments for how users should check for this specific error.

Reply:
Done